indico/modules/events/templates/management/_affiliation.html

```diff
@@ -0,0 +1,19 @@
+{%- macro render_affiliation(person) -%}
+    {% set details = person.affiliation_details %}
+    {%- if details -%}
+        {%- set details = person.affiliation_details -%}
+        {%- set uuid = uuid() -%}
+        <span id="affiliation-popup-container-{{ uuid }}">{# -#}
```

Comment:
in most other places we use `{#--#}` (but of course yours works as well since there's only whitespace one the right side that needs to be stripped)